### PR TITLE
CSS Basics pokemon page exercise

### PR DESCRIPTION
**Pull request by CristobalCaroF**

### CODE REVIEW

**Review by M6K4N9R — COMMENTED** [review]
I guess it is also good.

File: css-basics/css-basics_pokemon-page/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+- A fancy add on: Add a solid 5px border with the color "orange" that is only visible on the left. Hint: Use "border-left". :)
+- Bonus:  The left border of the second section which contains a list of favourite Pokémon should have the color "gold". Hint: You may need a second CSS rule.
+*/
+
+section {
+  /* your solution */
+  background-color: white;
+  padding: 20px;
+  margin: 10px;
+  border-left: solid orange;
```
Comment (janavoigt) [inline css-basics/css-basics_pokemon-page/css/styles.css:64]: add - border-left: 5px

File: css-basics/css-basics_pokemon-page/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- Add a padding of 10px
+- Center the text of the heading
+*/
+
+.title {
+  /* your solution */
+  background-color: white;
+  margin-top: 0;
+  padding: 10px;
+  text-align: center;
```
Comment (janavoigt) [inline css-basics/css-basics_pokemon-page/css/styles.css:46]: border left and border right 3px solid with the color "gold" of the tittle only

File: css-basics/css-basics_pokemon-page/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  padding: 20px;
+  margin: 10px;
+  border-left: solid orange;
+}
+
+.favourites {
+  /* background-color: white;
+  padding: 20px;
+  margin: 10px; */
+  border-left: solid gold;
```
Comment (janavoigt) [inline css-basics/css-basics_pokemon-page/css/styles.css:71]: add - border-left: 5px

**Review by janavoigt — COMMENTED** [review]
hope this helps! :)

**Pull request merged**